website/sidebars.json

```diff
@@ -22,7 +22,8 @@
     ],
     "The Basics (MacOS)": [
       "rnm-getting-started",
-      "rnm-dependencies"
+      "rnm-dependencies",
+      "rn-macos-hermes"

```

Comment:
Given that the doc is talking about both Windows and macOS (with Windows content to come in later), should this be under a common header and not under the macOS docs?

Reply:
As far as I know we don't have a common header... but I can make one

Reply:
moved to a created "experimental" section.